android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/av/CustomHeadersOkHttpDataSourceFactory.java

```diff
@@ -10,8 +10,8 @@
 
 import java.util.Map;
 
-import expolib_v1.okhttp3.CacheControl;
-import expolib_v1.okhttp3.Call;
+import okhttp3.CacheControl;
```

Comment:
I think we could remove this file altogether (`expo-av` and `expoview` will use the same `okhttp` now) 🤔